### ISSUE TITLE
ESS

### ISSUE THREAD

**Issue by mlindsk** [mikldk/spartropy#1]
Hi mikl,

Interesting project. Could be very useful in ess as a substitute for [ess entropy](https://github.com/mlindsk/ess/blob/master/R/api_entropy.R)

which is a naive R approach.

/Mads

**Comment by mikldk** [comment]
Yes, that might me. Currently it's just because I need to calculate entropy for sparse high dimensional data containing many zeros. So I am experimenting with this to avoid having to explicitly constructing the sparse tables.

**Comment by mikldk** [comment]
@mlindsk : Update to API ([vignette](https://mikldk.github.io/spartropy/articles/introduction.html#example)) - but more might happen. Also, a few non-exported low-level functions.

**Comment by mlindsk** [comment]
I'll make a benchmark against ESS at some point ✌️Just to verify that ESS should rely on spartropy in the future.
